generation/new_client/README.md

```diff
@@ -137,10 +137,20 @@ The example in this README uses AlloyDB's [Cloud Drop](https://github.com/google
 ### API short name
 
 For convenience of the subsequent commands, define a variable for API short name.
+This value will be used by default to generate the following:
+* `distribution_name`
+* `destination-name`
+* `proto_path`
```

Comment:
Reading the section starting from line 154, I couldn't get how the default value would be constructed for proto-path using api_shortname. Would you clarify that?